components/clp-package-utils/clp_package_utils/controller.py

```
@@ -0,0 +1,773 @@
+        )
+        logger.info("CLP is started.")
+
+    def stop(self) -> None:
+        """
+        Stops CLP components deployed via Docker Compose.
+
+        :raise: Propagates `subprocess.run`'s exceptions.
+        """
+        try:
+            check_docker_dependencies(should_compose_run=True, project_name=self._project_name)
+        except EnvironmentError as e:
```

Ruff (0.14.0)

Replace aliased errors with `OSError`

Replace `EnvironmentError` with builtin `OSError`

(UP024)